lib/include/openamp/rpmsg.h

```diff
@@ -142,6 +149,21 @@ struct rpmsg_device {
 	bool support_ns;
 };
 
+/**
+ * @brief Increase the endpoint reference count
```

Comment:
Add  ` @internal` label as it should not be used by user.

Please also add a sentence to explain that the function should be called under `rdev->lock` protection
the comment is also valid for `rpmsg_decref`